gym-unity/gym_unity/envs/__init__.py

```diff
@@ -120,7 +129,7 @@ def __init__(
             high = np.array([1] * self.group_spec.action_shape)
             self._action_space = spaces.Box(-high, high, dtype=np.float32)
         high = np.array([np.inf] * self._get_vec_obs_size())
-        if self.use_visual:
+        if self._get_n_vis_obs() == 1:
```

Comment:
This should be 
```suggestion
        if self._get_n_vis_obs() > 0 and not self. _allow_multiple_obs:
```
Since we return a single visual obs only if allow_multiple_obs=True